src/object_support_segmentation.cpp

```diff
@@ -89,6 +89,15 @@ bool ObjectSupportSegmentation::segment(
 {
   RCLCPP_INFO(LOGGER, "object support segmentation starting...");
 
+  if (cloud->empty()) {
+    RCLCPP_ERROR(LOGGER, "Input pointcloud is empty.");
+    return false;
+  }
+  if (cloud->is_dense) {
+    RCLCPP_ERROR(LOGGER, "Input pointcloud is dense. Care removing NaN.");
```

Comment:
What does "Care removing NaN" actually mean? Also - is this actually needed (are you running the node as-is, or reusing this code somewhere else - because either the range filter converts to non-dense in the node OR this always hits?)